x-pack/test/detection_engine_api_integration/security_and_spaces/tests/create_threat_matching.ts

```diff
@@ -72,7 +72,13 @@ export default ({ getService }: FtrProviderContext) => {
           supertest,
           getCreateThreatMatchRulesSchemaMock('rule-1', true)
         );
-        await waitForRuleSuccessOrStatus(supertest, ruleResponse.id);
+
+        // expecting a 'partial failure' status because the threat match rule mock
+        // does not have any indices listed in the rule definition so the rule will
+        // technically be querying ".kibana_1", ".kibana_task_manager_1", and ".security"
+        // as those are the only indices defined.
+        // This mocked rule should be updated to query against an index like auditbeat-*
+        await waitForRuleSuccessOrStatus(supertest, ruleResponse.id, 'partial failure');
```

Comment:
I would move this to the tests below and change it to use the audit beat data like the ones below and have it still expect the 'succeeded'.

You can change it to use the same format as one of the tests below this. Otherwise we will start to have unusual tests that say are saying things like, "should create a single rule with a rule_id and validate it ran successfully" but instead of checking it ran successfully we only check if it ran with a partial failure when we want to ensure we do detect that it has the succeeded